### ISSUE TITLE
Multiple templates not supported

### ISSUE THREAD

**Issue by smaren** [gajus/eslint-plugin-jsdoc#622]
## Expected behavior

Multiple template types should be allowed with `settings.jsdoc.mode = "closure"`, i.e. `@template T, R`, see https://github.com/google/closure-compiler/wiki/Generic-Types#multiple-template-types

## Actual behavior

Multiple templated types trigger a warning: `warning  Syntax error in namepath: T,  jsdoc/valid-types`

## ESLint Config

```js
"plugins": ["jsdoc"],
"rules": {
  "jsdoc/valid-types": 1
}
"settings": {
  "jsdoc": {
    "mode": "closure"
  }
}
```

## ESLint sample

```js
/**
 * @template T, R
 * @param {function(!T): !R} parser
 * @return {function(!Array<!T>): !Array<!R>}
 */
parseArray = function(parser) {
    return function(array) {
        return array.map(parser);
    };
};
```
`warning  Syntax error in namepath: T,  jsdoc/valid-types`

## Environment

- Node version:  v10.21.0
- ESLint version  v7.6.0
- `eslint-plugin-jsdoc` version: v30.2.1
